linker/shared/src/main/scala/org/scalajs/linker/standard/CoreSpec.scala

```diff
@@ -96,6 +96,9 @@ final class CoreSpec private (
       targetIsWebAssembly
     )
   }
+
+  lazy val linkTimeProperties = new LinkTimeProperties(
+      semantics, esFeatures, targetIsWebAssembly)
```

Comment:
Oops, I don't know how Scala lets this through, but since `LinkTimeProperties` is now `private[linker]`, this lazy val should also be `private[linker]`:
```suggestion
  private[linker] lazy val linkTimeProperties = new LinkTimeProperties(
      semantics, esFeatures, targetIsWebAssembly)
```

Reply:
Thanks, not sure why it is allowed 🤔 but indeed it's should be `private[linker]`